pkg/common/helpers.go

```diff
@@ -49,3 +64,67 @@ func IsExpired(startTime int64, ttlTime int64) bool {
 	}
 	return false
 }
+
+// NewWFClient -- declare new workflow client
+func NewWFClient(cfg *rest.Config) wfv1versioned.Interface {
+	cli, err := wfv1versioned.NewForConfig(cfg)
+	if err != nil {
+		panic(err)
```

Comment:
No panics!

Reply:
Can't we just have 1 injection call `wfv1versioned.NewForConfigOrDie(ctrl.GetConfig())` why do we need all these helpers?

Reply:
If you really do not like them, I can remove it. But, it helps developer to understand the details.

Reply:
f91a53ffc8d59d0bc3891a8d494fa2ed5eafb406